docs/source/gbd2017_models/causes/neoplasms/cervical_cancer/cancer_model.rst

```diff
@@ -238,8 +243,8 @@ State and Transition Data Tables
      - weighted average of cervical cancer disability weight over all sequelae including ids s_282, s_283, s_284, s_285
    * - ICC, C_hrHPV
      - prevalence (prev_ICC_and_C_hrHPV)
-     - :math:`\frac{\text{prev_c432}\times\text{RR_hrHPV}}{\text{RR_hrHPV}+1}`
-     - used only at initialization
+     - :math:`\text{prev_c432}\times\text{PAF}\times\frac{\text{RR_hrHPV}}{\text{RR_hrHPV}-1}`
```

Comment:
I expected this to be prev_c432*(1-PAF)*RR

Reply:
I revised this equation because one extreme situation doesn't work. For example if prevalence of hrHPV = 0, then PAF = 0, the prevalence of ICC with HPV would be prev_c432 * (1 - 0) * RR = prev_c432 *RR rather than zero.

Reply:
Hmm, yeah, I didn't think this out carefully enough.  What do you think of simply making it prev_c432*PAF?